File: Conscious Python - OOP/step 03. Наследование/3.2.01.py
'''

    Создайте два класса Minecraft и Roblox. Класс Roblox - будет дочерним классом Minecraft.
    В классе Minecraft создайте метод hello_creeper, который будет выводить на экран фразу - "Hello, Creeper!".
    В классе Roblox создайте метод hello_all, который будет вызывать метод hello_сreeper. Метод hello_all также будет выводить на экран фразу "Hello, Pozzy!" с помощью print.
    Создайте экземпляр hello класса Roblox.
    Вызовите метод hello_all, через экземпляр hello. Функцию print не нужно использовать.

В итоге, программа будет выводить на экран две фразы, как указано в примере ниже. 

Sample Input:

Sample Output:

Hello, Creeper!
Hello, Pozzy!


'''

class Minecraft:
    def hello_creeper(self):
        print('Hello, Creeper!')

class Roblox(Minecraft):
    def hello_all(self):
        super().hello_creeper()
        print('Hello, Pozzy!')
hello = Roblox()
hello.hello_all()

File: Diving into JavaScript: For Beginners/step03/3.02/3.2.01.js
/*

Напишите программу, которая проверяет, является ли введенное пользователем число четным, 
и выводит сообщение "Число четное", если число четное или "Число нечетное", если число нечетное.

Sample Input 1:

1

Sample Output 1:

Число нечетное

Sample Input 2:

0

Sample Output 2:

Число четное

Sample Input 3:

156

Sample Output 3:

Число четное

*/

let number = Number(prompt());

if (number % 2 === 0) {
    console.log('Число четное');
} else {
    console.log('Число нечетное');
}
